### ISSUE TITLE
Provide a sample ACME partner report using these fields

### ISSUE THREAD

**Issue by MainStreamDanny** [gpsimpact/relational_gotv#59]
As soon as we can, I'd like to provide Brandi and Lindsay (our new IVE hire) with an example what partners will see, one that is more relevant to what we are actually doing. Ideally this would be one of the first things after forking a MainStream specific build.

Fields I can imagine partners wanting:

ambassador name (first, last?)
ambassador contact info (email, phone?)
ambassador's points earned to date (points, date)
ambassador's voter 1 (name, address, voter?, vbm?, tasks done of total tasks, tasks done (reg, vbm, etc.), task not done (voted, met rep, etc.), voted?)
ambassador's voter 2 (ibid.)
...etc...
ambassador's voter x (ibid.)

Huh. That feels like it isn't a flat file spreadsheet... ideas? It would have to be something like:

amb1name / amb1info / amb1pts / v1name / v1add / v1voter / v1vbm / ... / v2name / v2add / ...etc.
amb2name / amb2info / amb2pts / v1name / v1add / v1voter / v1vbm / ... / v2name / v2add / ...etc.

But that would be really hard to parse. You could compare ambassadors by points (crude but probably reasonable), but it would be really hard to get a handle on how the voters were doing. I think maybe one list of ambassador (in case they want a contact list), that would also points listed. Then one list of voters for each ambassador. Several downloads, but I don't know how to get them into one file and be reasonably understandable.

Thoughts?

**Comment by jakelowen** [comment]
Need: Ambassador level rollup (# of points, # of voters, etc) per org

Detail view of voters with ambassador names

For above system analgues for danny / lindsay
Also partner rollup (similar to ambassador rollup)
